libs/utils/src/compressed_tallies.ts

```diff
@@ -222,32 +221,32 @@ function getContestTalliesForCompressedContest(
   }
 }
 
+/**
```

Comment:
did you test this part manually? I would recommend it for sanity even though its annoying

Reply:
The manual test I did for this was:

- Scan some ballots in VxScan
- Print report, check results match expected
- Export results to card and print report on VxMark, check results match expected

I only did this for the primary all precincts case, assuming that would catch most issues. Did not explicitly check for primary single precinct case.

Reply:
sounds good